src/main/resources/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder/global.jelly

```diff
@@ -2,16 +2,24 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 
     <f:section title="AWS Device Farm">
-        <!--
+        <f:block>
+            <div>
+                We support Role arn or Access and Secret Key as means to access Device Farm.Please provide details for one of these below and click on Validate Credentials to ensure they work.
```

Comment:
We support Role ARN or Access and Secret Key as means to access Device Farm. <add space>Please

Reply:
Changed